1-Draft/RFCNNNN-Simplify-Module-Manifests.md

```diff
@@ -0,0 +1,170 @@
+---
+RFC: RFCnnnn
+Author: Kirk Munro
+Status: Draft
+SupercededBy: 
+Version: 1.0
+Area: Module Manifests
+Comments Due: July 15, 2019
+Plan to implement: Yes
+---
+
+# Title
+
+When PowerShell module support first came out in PowerShell 2.0, module manifests had a specific list of keys that could be used to define the manifest. Additional keys were added in PowerShell 3.0, but that hurt the end user experience because at the time, if you loaded a module with a manifest containing keys that were not recognized/supported in your version of PowerShell, PowerShell would return an unhelpful error about the module keys being invalid instead of checking for the minimum required version of PowerShell and returning an error indicating that the module only supports that version or later.
+
+Beyond PowerShell 3.0, keys started being added to a PSData subsection of the PrivateData area of modules. You can see these keys in use by the PowerShell Gallery. That approach has its own challenges, because now keys are defined in a hierarchy, which adds complexity and should not be necessary.
```

Comment:
Please add examples (links to examples) - this helps reviewers.

Reply:
Will submit an update shortly.